airbyte-config/models/src/main/resources/types/JobSyncConfig.yaml

```diff
@@ -6,18 +6,24 @@ description: job sync config
 type: object
 additionalProperties: false
 required:
-  - sourceConnection
-  - destinationConnection
-  - standardSync
+  - sourceConfiguration
```

Comment:
This is just a json object containing the configuration fields in specified in the spec?

Reply:
yes